CemrgApp/Plugins/kcl.cemrgapp.easi/src/internal/EASIView.cpp

```diff
@@ -577,7 +577,7 @@ void EASIView::ConfrmSITE() {
 void EASIView::Simulation() {
 
     std::string line;
-    ifstream file("/home/or15/Work/Strain/ResolutionStudy/paths.txt");
+    std::ifstream file("/home/or15/Work/Strain/ResolutionStudy/paths.txt");
```

Comment:
This plugin is not in use, right? It has hardcoded paths to my old home directory. It should probably be checked.

Reply:
This plugin is not used. It could be that we could just get rid of it or add something like a user-defined path. In my opinion, since it's not being used at the moment, we could leave that for another PR.

Reply:
I’ve created #81 to track this.